apps/camera_pipe/camera_pipe_generator.cpp

```diff
@@ -306,9 +323,8 @@ Func CameraPipe::build() {
 
     Func denoised = hot_pixel_suppression(shifted);
     Func deinterleaved = deinterleave(denoised);
-    Func demosaiced; Scheduler demosaiced_scheduler;
-    std::tie(demosaiced, demosaiced_scheduler) = demosaic(deinterleaved);
-    Func corrected = color_correct(demosaiced);
+    auto demosaiced = Demosaic::apply(*this, deinterleaved);
```

Comment:
What if it's also the same method on the JITGeneratorContext. So JITGeneratorContext acts like a parent generator for cases where you don't have one.